client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/hosting-badge/style.scss

```diff
@@ -0,0 +1,10 @@
+.migration-instructions-hosting-badge {
+	width: fit-content;
+	margin: 0 0 20px auto;
+	padding: 0 10px;
+	font-size: 0.75rem;
+	font-weight: 500;
+	line-height: 20px;
```

Comment:
As the `font-size` is in rem, wondering if the `line-height` should also be in rem?

Reply:
Agree that the line-height should also be relative since the font-size is.

In a search all, it seems the standard would be the multiplier, like `line-height: 1.5`. 🤔
But I didn't find anything confirming the standard.

Reply:
Makes sense. It was updated in https://github.com/Automattic/wp-calypso/pull/92215/commits/37e870f02f822c601b480bb74dad2d6548c82092. 👍